dockerclient.go

```diff
@@ -262,6 +262,56 @@ func (client *DockerClient) readJSONStream(stream io.ReadCloser, decode func(*js
 	return resultChan
 }
 
+func (client *DockerClient) ExecCreate(config *ExecConfig) (string, error) {
+	data, err := json.Marshal(config)
+	if err != nil {
+		return "", err
+	}
+	uri := fmt.Sprintf("/containers/%s/exec", config.Container)
```

Comment:
hm, should we be putting the `APIVersion` in the URI? it seems to be standard practice, looking at other requests in this file